README.md

```diff
@@ -10,7 +10,7 @@ Latest versions:
 
 | Used tag | Effective version |
 | :---: | :---: |
-| `v2` | [`v2.2.2`](https://github.com/Materials-Consortia/optimade-validator-action/releases/tag/v2.2.2)
+| `v2` | [`v2.3.0`](https://github.com/Materials-Consortia/optimade-validator-action/releases/tag/v2.3.0)
```

Comment:
This will be automagically updated via the `invoke` task the updates version.
```suggestion
| `v2` | [`v2.2.2`](https://github.com/Materials-Consortia/optimade-validator-action/releases/tag/v2.2.2)
```